site/cds_rdm/resolver.py

```diff
@@ -0,0 +1,14 @@
+from invenio_pidstore.models import PersistentIdentifier
+from invenio_pidstore.resolver import Resolver
+
+
+def get_pid_by_legacy_recid(pid_value):
```

Comment:
nitpick: could the argument be `legacy_pid_value` as there's visual repetition of `pid_value` in the function which could be confusing